src/Cards.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Cards = (props) => {
```

Comment:
The naming here implies that the component shows multiple cards, but it's only showing 1. I would normally call this `Card` instead.